Content.Server/Store/Systems/StoreSystem.Refund.cs

```diff
@@ -16,7 +16,7 @@ private void InitializeRefund()
 
     private void OnEntityRemoved(EntityUid uid, StoreRefundComponent component, EntRemovedFromContainerMessage args)
     {
-        if (component.StoreEntity == null || _actions.TryGetActionData(uid, out _) || !TryComp<StoreComponent>(component.StoreEntity.Value, out var storeComp))
+        if (component.StoreEntity == null || _actions.TryGetActionData(uid, out _, false) || !TryComp<StoreComponent>(component.StoreEntity.Value, out var storeComp))
```

Comment:
Is throwing on item purchases in master branch now! Looks wierd. So ignore it? or tests going to break <_<